parser/prism/Translator.cc

```diff
@@ -832,8 +832,10 @@ unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
             unreachable("The `it` keyword was introduced in Ruby 3.4, which isn't supported by Sorbet yet.");
         }
         case PM_KEYWORD_HASH_NODE: { // A hash of keyword arguments, like `foo(a: 1, b: 2)`
-            auto usedForKeywordArgs = true;
-            return translateHash(node, down_cast<pm_keyword_hash_node>(node)->elements, usedForKeywordArgs);
+            auto kvPairs = translateKeyValuePairs(down_cast<pm_keyword_hash_node>(node)->elements);
+            auto isKwargs = absl::c_all_of(kvPairs, [](const auto &node) { return isKeywordHashElement(node.get()); });
```

Comment:
First time seeing code like this. What does it do?
Also, I think it'd be easier to read if we can expand it to 3 lines:

```suggestion
            auto isKwargs = absl::c_all_of(kvPairs, [](const auto &node) { 
                return isKeywordHashElement(node.get()); 
            });
```